plugins/modules/eos_route_maps.py

```diff
@@ -0,0 +1,1404 @@
+                          type: str
+                          choices: ["equal", "ration"]
+                    none:
+                      description: No attribute.
+                      type: bool
+                    rt:
+                      description: Route target extended community
+                      type: dict
+                      suboptions: &params01
+                        vpn:
+                          description: VPN extended community.
+                          type: str
+                        additive:
+                          description: Add to the existing community.
+                          type: bool
+                        delete:
+                          description: Delete matching communities.
+                          type: bool
+                    soo:
+                      description: Site-of-Origin extended community.
+                      type: dict
+                      suboptions: *params01
+                ip:
+                  description: Set IP specific information.
+                  type: dict
+                  suboptions: &params02
+                    address:
+                      description: next hop address.
+                      type: str
+                    unchanged:
+                      description: Keep the next hop when advertising to eBGP peer
+                      type: bool
+                    peer_address:
+                      description: Use BGP peering addr as next-hop.
+                      type: bool
+                ipv6:
+                  description: Set IPv6 specific information.
+                  type: dict
+                  suboptions: *params02
+                isis_level:
+                  description: IS-IS level.
+                  type: str
+                local_preference:
+                  description: BGP local preference.
+                  type: int
+                metric:
+                  description: Route metric.
+                  type: dict
+                  suboptions:
+                    igp_param:
+                      description: IGP parameter
+                      type: str
+                      choices: ['igp-metric', 'igp-nexthop-cost']
+                    add:
+                      description: Add igp-metric / igp-nexthop-cost
+                      type: str
+                      choices: ['igp-metric', 'igp-nexthop-cost']
+                    value:
+                      description: metric value to add or subtract(with +/- sign).
+                      type: str
+                metric_type:
+                  description: Route metric type.
+                  type: str
+                  choices: ["type-1", "type-2"]
+                nexthop:
+                  description: Route next hop.
+                  type: dict
+                  suboptions:
+                    value:
+                      description: IGP metric value.
+                      type: int
```

Comment:
can u plz check if this also supports list of int

Reply:
It takes a single integer.